package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "epm-spec",
```

Comment:
I'm trying to get everything renamed to `ethpm` to reduce any name collisions. Can we rename to `ethpm-spec`? This should likely coincide with a renaming of the repo as well.